conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -0,0 +1,132 @@
+                        ),
+                        usage=argparse.SUPPRESS,
+                        add_help=False,
+                        formatter_class=formatter)
+        hostfactory_create = create_cmd.add_argument_group(
+            title=title_formatter("Options"))
+        hostfactory_create.add_argument('-h', '--help', action='help',
+                                        help='Display help screen and exit')
+        return create_cmd.add_subparsers(title="Subcommand", dest='action')
+
+    @staticmethod
+    def _add_hostfactory_create_token(menu: ArgparseWrapper):
+        hostfactory_create_token_name = 'token - Create token/s for hosts ' \
+                                        'with restrictions'
+        hostfactory_create_token_usage = 'conjur [global options] hostfactory ' \
+                                         'create token [options] [args]'
+
+        hostfactory_create_subcommand_parser = menu \
+            .add_parser(name="token",
+                        help='Create token/s for hosts with restrictions',
+                        description=command_description(
+                            hostfactory_create_token_name, hostfactory_create_token_usage),
+                        epilog=command_epilog(
+                            'conjur hostfactory create token --hostfactoryid my_factory '
+                            '--cidr 10.10.1.2/31 '
+                            '--duration-days 2\t\t '
+                            'Creates one or more identical tokens for hosts with restrictions\t\t',
+                            command='token',
+                        ),
+                        usage=argparse.SUPPRESS,
+                        add_help=False,
+                        formatter_class=formatter)
+        # Options
+        create_token = hostfactory_create_subcommand_parser.add_argument_group(
+            title=title_formatter("Options"))
+        create_token.add_argument('-action_type', default='create_token', help=argparse.SUPPRESS)
+        create_token.add_argument('-i', '--hostfactoryid', metavar='VALUE',
+                                  help='(Mandatory) the ID of the host factory.')
+        create_token.add_argument('--cidr', metavar='VALUE',
+                                  help='(Optional) the CIDR address that contains '
+                                       'all IPs that can use this token to create hosts. '
+                                       'You can specify multiple cidr, '
+                                       'separated by commas (for example '
+                                       '--cidr "10.0.10.0/24,'
+                                       '10.0.11.1/32,10.0.20.0/24")')
+        create_token.add_argument('-d', '--duration-days', metavar='VALUE', type=int,
+                                  help='(Optional) the number of days the token will be valid.')
+        create_token.add_argument('-H', '--duration-hours', metavar='VALUE', type=int,
```

Comment:
was anything decided here regarding the -H?

Reply:
@sigalsax not that I know of